app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+
+class HomeController extends Controller
+{
+
+  public function index(){
+    $user =Auth::user();
+
+
+    $folder =$user->folders()->first();
+
+
+    if(is_null($folder)) {
+      return view('home');
+    }
+
+    return redirect()->route('tasks.index',[
+      'id' => $folder->id,
+    ]);
+  }
+}
```

Comment:
このチュートリアルは一応個人プロジェクトという扱いでいいかなと思っているんですが、お仕事で書くコードはPSR2に則った形でやることが多いかなと思います。PSR-2ではインデントの形式を指定しています
> Code MUST use 4 spaces for indenting, not tabs.

https://www.php-fig.org/psr/psr-2/

---

（PSRの話）
`PSR-${数字}` というのはPHPの有志が「こういう感じでやるのがいいよね」と決めているものなので絶対に守らなければいけないわけではないです。ただ、この形式を守ることでエコシステムの恩恵を受けやすくなります。
たとえば、Laravelを使用した開発ではほぼ必ず依存（ライブラリ）管理ツールである[Composer](https://getcomposer.org/) を使用すると思いますが、Composerは[PSR-4](https://www.php-fig.org/psr/psr-4/)に則ったディレクトリ・ファイル配置のautoloadをサポートしています。

（PSRの話終わり）

---

なのでPHPのコードは基本4スペースでインデントしちゃいましょう〜
規約とか「こうしましょう」というのがチームで決まっていたらそれを優先で大丈夫です :pray:

```suggestion
class HomeController extends Controller
{
    public function index() {
        $user = Auth::user();
        $folder = $user->folders()->first();

        if (is_null($folder)) {
            return view('home');
        }

        return redirect()->route('tasks.index', [
            'id' => $folder->id,
        ]);
    }
}
```

Reply:
atom にインデントフォーマットを導入